homeassistant/components/switchbot/sensor.py

```diff
@@ -0,0 +1,91 @@
+"""Support for SwitchBot sensors."""
+from __future__ import annotations
+
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_MAC,
+    CONF_NAME,
+    DEVICE_CLASS_BATTERY,
+    DEVICE_CLASS_ILLUMINANCE,
+    DEVICE_CLASS_SIGNAL_STRENGTH,
+    PERCENTAGE,
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DATA_COORDINATOR, DOMAIN
+from .coordinator import SwitchbotDataUpdateCoordinator
+from .entity import SwitchbotEntity
+
+PARALLEL_UPDATES = 1
+
+SENSOR_TYPES: dict[str, SensorEntityDescription] = {
+    "rssi": SensorEntityDescription(
```

Comment:
Usually we disable these type of sensors by default and let the user enable the entity if they need it since they tend to generate a large number of state changes and are only useful when debugging a connection issue

Reply:
https://developers.home-assistant.io/docs/core/entity/#advanced-properties

entity_registry_enabled_default

Reply:
Done.